packages/create-svelte/package.json

```
@@ -33,7 +33,8 @@
 		"lint": "eslint --ignore-path .gitignore --ignore-path ../../.gitignore \"./*.js\" && npm run check-format",
 		"format": "npm run check-format -- --write",
 		"check-format": "prettier --check . --config ../../.prettierrc --ignore-path ../../.gitignore --ignore-path .gitignore --plugin prettier-plugin-svelte --plugin-search-dir=.",
-		"prepublishOnly": "npm run build"
+		"prepublishOnly": "npm run build",
+		"postpublish": "./scripts/update-template-repo.sh"
```

```suggestion
		"postpublish": "bash ./scripts/update-template-repo.sh"
```

Do we need a leading `bash` command here? IIRC it's needed in Ubuntu (CI).